docker.go

```diff
@@ -164,6 +167,18 @@ func getRoleArnFromEnv(env []string) (role roleArn, policy string, err error) {
 			}
 		} else if v[0] == "IAM_POLICY" && len(v) > 1 {
 			policy = strings.TrimSpace(v[1])
+		} else if v[0] == "IAM_ROLE_NAME" && len(v) > 1 {
+			roleName = strings.TrimSpace(v[1])
+		} else if v[0] == "AWS_ACCOUNT_ID" && len(v) > 1 {
```

Comment:
@dump247 I'll add some functionality to automatically detect the account id through curling the ec2 metadata.